src/keys/utils.ts

```diff
@@ -27,34 +29,85 @@ import {
   pkcs5,
   util as forgeUtil,
 } from 'node-forge';
+import * as bip39 from 'bip39';
 import * as keysErrors from './errors';
 import config from '../config';
 import * as utils from '../utils';
-import { promisify } from '../utils';
+import { never, promisify } from '../utils';
 import { makeNodeId } from '../nodes/utils';
+import * as networkUtils from '../network/utils';
+import { Status } from '../status';
+import * as polykeyErrors from '../errors';
+
+bip39.setDefaultWordlist('english');
 
 const ivSize = 16;
 const authTagSize = 16;
 
+async function verifyMnemonicMatchesNode(
```

Comment:
I think this function should be moved into the `KeyManager`. This has alot of side-effectful logic that should be in key manager. Where is it being used?

Reply:
Fixed.